hubblestack/extmods/modules/pulsar.py

```diff
@@ -634,7 +634,7 @@ def mark(self,name):
 @hubble_status.watch
 def process(configfile='salt://hubblestack_pulsar/hubblestack_pulsar_config.yaml',
             verbose=False):
-    """
+    r"""
```

Comment:
false. That is because there is an escaped character in the docstring. the 'r' is required for it to be properly parsed and formatted (at least by linters).